subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/LintCheck.kt

```diff
@@ -14,19 +14,28 @@ import org.gradle.kotlin.dsl.invoke
 class LintCheck(context: String, name: String) : SuppressibleBuildStep(context, name),
     ImpactAnalysisAwareBuildStep by ImpactAnalysisAwareBuildStep.Impl() {
 
+    var slackChannelForAlerts = ""
+
     override fun registerTask(project: Project, rootTask: TaskProvider<out Task>) {
         require(project.isAndroidApp()) {
             "Lint check can be applied only to android application modules"
         }
+
         if (project.buildEnvironment !is BuildEnvironment.CI) return
+
         if (useImpactAnalysis && !project.internalModule.isModified()) return
 
         project.pluginManager.withPlugin("com.avito.android.lint-report") {
 
             val lintReports = project.tasks.withType<LintReportTask>()
+
             lintReports.configureEach {
+                if (slackChannelForAlerts.isNotBlank()) {
```

Comment:
If `isBlank` throw IllegalStateException

Reply:
it's fine to run this task without slack integration;
it should be different tasks after all i think, but it's too early to decide it; 
research about how we should run lint comes before